src/Containers/PaletteForm/PaletteForm.js

```diff
@@ -1,17 +1,84 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { postPalette } from '../../Thunks/postPalette';
+import { fetchProjectPalettes } from '../../Thunks/fetchProjectPalettes';
 
 export class PaletteForm extends Component{
+  constructor() {
+    super();
+    this.state = {
+      name: '',
+      projectId: ''
+    };
+  };
+
+  createPalette = () => {
+    const { name } = this.state;
+    const {currentPalette} = this.props
+    let paletteToAdd = currentPalette.reduce((acc, color, index) => {
```

Comment:
Great work here creating a palette with the properties color1, color2... from currentPalette!